snap/snapcraft.yaml

```diff
@@ -189,15 +189,20 @@ parts:
       # These are only included here until the OS snap stabilizes
       - libxml2
       - libpng12-0
+      - libmagickwand-6.q16-2
+      - libmagickcore-6.q16-2
```

Comment:
Have you tried leaving these off and seeing if the snapcraft CLI pulls them over for you?

Reply:
No, I haven't, but I will now.